include/message_filters/subscriber.h

```diff
@@ -113,12 +133,33 @@ class Subscriber : public SubscriberBase, public SimpleFilter<M>
    * \param topic The topic to subscribe to.
    * \param qos (optional) The rmw qos profile to use to subscribe
    */
-  Subscriber(rclcpp::Node::SharedPtr node, const std::string& topic, const rmw_qos_profile_t qos = rmw_qos_profile_default)
+  Subscriber(rclcpp::Node::SharedPtr node, const std::string& topic, const rmw_qos_profile_t qos = rmw_qos_profile_default) 
```

Comment:
Probably unintentional trailing whitespace.

```suggestion
  Subscriber(rclcpp::Node::SharedPtr node, const std::string& topic, const rmw_qos_profile_t qos = rmw_qos_profile_default)
```

Reply:
Fixed